### PR TITLE
implements end date not in future validation

### PR DESCRIPTION
**Pull request by BlakeDraper**

### CODE REVIEW

File: src/app/add-event-location/add-event-location.component.ts [hunk 1]
```diff
@@ -136,49 +151,59 @@ export class AddEventLocationComponent implements OnInit {
...
+        environmental_factors: "",
+        clinical_signs: "",
+        comment: "",
+        // used only to capture event_type from event - not part of eventlocation record
+        event_type: null,
+        new_location_species: this.formBuilder.array([
+          this.initLocationSpecies(),
+        ]),
+        new_location_contacts: this.formBuilder.array([
+          // this.initLocationContacts()
```
Comment (lprivette) [inline src/app/add-event-location/add-event-location.component.ts:196]: is this supposed to be commented out?

Comment (lprivette) [reply src/app/add-event-location/add-event-location.component.ts:196]: Nm I see it being called in another function to create the control

Comment (BlakeDraper) [reply src/app/add-event-location/add-event-location.component.ts:196]: Yeah, long story with the form creation stuff, but I'd like the comment to stay for now.

**Review by lprivette — APPROVED** [review]

**Comment by lprivette** [comment]
nice work!

**Pull request merged**